library/src/window_impl.cxx

```diff
@@ -549,9 +549,9 @@ void window_impl::SetImporter(vtkF3DMetaImporter* importer)
 }
 
 //----------------------------------------------------------------------------
-void window_impl::SetCachePath(const std::string& cachePath)
+void window_impl::SetCachePath(const std::filesystem::path& cachePath)
 {
-  this->Internals->CachePath = cachePath;
+  this->Internals->CachePath = cachePath.string();
```

Comment:
Maybe `CachePath` should be a fs::path too?

Reply:
indeed